zerver/lib/email_mirror.py

```diff
@@ -28,11 +32,13 @@ def redact_stream(error_message):
     return error_message
 
 def report_to_zulip(error_message):
+    # type: (str) -> None
     error_stream = Stream.objects.get(name="errors", realm__domain=settings.ADMIN_DOMAIN)
     send_zulip(error_stream, "email mirror error",
                """~~~\n%s\n~~~""" % (error_message,))
 
 def log_and_report(email_message, error_message, debug_info):
```

Comment:
After looking at `process_message`, I thought that the `email_message` parameter in `log_and_report` should be annotated as `message`. What do you think?